camp/JHall/leftJoystickToTalon/src/main/java/frc/robot/subsystems/TalonMotorSubsystem.java

```diff
@@ -0,0 +1,86 @@
+package frc.robot.subsystems;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.StatusCode;
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.configs.Slot1Configs;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.VelocityVoltage;
+import com.ctre.phoenix6.controls.VoltageOut;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+
+import edu.wpi.first.wpilibj.motorcontrol.Talon;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+//ignore for now
+import frc.robot.Constants;
+
+
+public class TalonMotorSubsystem extends SubsystemBase{
+    private final TalonFX motor;
+    private final VelocityVoltage velocityController = new VelocityVoltage(0);
+    private final VoltageOut voltageController = new VoltageOut(0);
+    private Slot0Configs motorRpmGains = new Slot0Configs().withKP(0.36).withKI(0.1).withKD(0.0075).withKV(0.10);
```

Comment:
It's OK to use magic numbers here, but you need to include a comment about where they came from. Add a comment that they are copied from Ted ShooterOutfeedMotor